go/vt/tabletserver/query_executor.go

```diff
@@ -131,7 +142,21 @@ func (qre *QueryExecutor) Execute() (reply *mproto.QueryResult, err error) {
 func (qre *QueryExecutor) Stream(sendReply func(*mproto.QueryResult) error) error {
 	qre.logStats.OriginalSQL = qre.query
 	qre.logStats.PlanType = qre.plan.PlanId.String()
-	defer qre.qe.queryServiceStats.QueryStats.Record(qre.plan.PlanId.String(), time.Now())
+
```

Comment:
Please add a type to the mapped Varz to differentiate between Stream V.S NonStream